### ISSUE TITLE
Is it possible to get FPS Sample Working in Unity 2020

### ISSUE THREAD

**Issue by albrightd** [Unity-Technologies/FPSSample#139]
I downloaded the asset and thought I set everything up right but kept getting errors about missing assets. Does anyone know if its possible to get it working and could you possibly point me in the right direction?

**Comment by abnegate** [comment]
If you downloaded a ZIP from github, large assets stored using git-lfs are _not_ included.

You'll need to ensure you have [git-lfs](https://git-lfs.github.com/) installed, then `git clone` the repo instead of download ZIP.

**Comment by SliceWire** [comment]
No, you cannot use it in unity 2020 or even 2019. The demo team promised that they would update it, but I believe they abandoned it. You have to clone it with lfs and then open it in unity 2018.3 or 2018.4.

**Comment by Jayveer722** [comment]
You can use but in switching it to unity editor 2020,2019or2021 it will take 10 days

**Comment by deviousDeckHand** [comment]
> You can use but in switching it to unity editor 2020,2019or2021 it will take 10 days

Do you know what to do/ how to get it into 2020? Or do you know where I can find a 2020 version of it?

**Comment by JairusSW** [comment]
I'm not sure... Someone was working on it, but it was abandoned also. :(

**Comment by baponkar** [comment]
I have downloaded by 'git clone' method but it is showing following errors:

Multiple precompiled assemblies with the same name ICSharpCode.NRefactory.dll included or the current platform. Only one assembly with the same name is allowed per platform. (B:/Unity_Projects/FPSSample/Packages/com.unity.render-pipelines.high-definition/Editor/Core/ShaderGenerator/ICSharpCode.NRefactory.dll)

Assembly 'Unity.Properties.Codegen' is a CodeGen assembly and cannot be Auto Referenced (Packages/com.unity.properties/Unity.Properties.Codegen/Unity.Properties.Codegen.asmdef)


for first error I have tried to remove .dll files and for next one I have tried to change  
 "autoReferenced": true, to  "autoReferenced": false in
(Packages/com.unity.properties/Unity.Properties.Codegen/Unity.Properties.Codegen.asmdef).

But it is still not working and showing same error.
What should I do to remove those two error?

**Comment by JairusSW** [comment]
What version of unity?

**Comment by baponkar** [comment]
@JairusSW 
Unity 2020.3.15f2 Personal

**Comment by JairusSW** [comment]
Okay, you have to use unity 2018. Follow the guides in the repo

**Comment by JairusSW** [comment]
It will work with 2018. Its abandoned, sooo, its not getting any updates

**Comment by m-pawar** [comment]
Well, I have a better problem than this, I am always unable to see scenes except for Bootstrapper in the Project Tool window
.
